<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolTabShortInfoBox</class>
 <widget class="QWidget" name="toolTabShortInfoBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="meshGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh</string>
   </property>
   <widget class="QGroupBox" name="boundBoxGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Bound Box</string>
    </property>
    <widget class="QLabel" name="xSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X Size= </string>
     </property>
    </widget>
    <widget class="QLabel" name="ySizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y Size= </string>
     </property>
    </widget>
    <widget class="QLabel" name="zSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z Size= </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="trianglesValueLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangles=</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Model matrix</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
